src/test/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Are test properties necessary now?

Reply:
CI checks do not pass without that and this step is  included in the requirements